pinot-core/src/test/java/com/linkedin/pinot/index/readerwriter/FixedByteSingleColumnMultiValueReaderWriterTest.java

```diff
@@ -62,4 +73,62 @@ public void testIntArray(final long seed)
     }
     readerWriter.close();
   }
+
+  public void testIntArrayFixedSize(int multiValuesPerRow, long seed)
+      throws IOException {
+    FixedByteSingleColumnMultiValueReaderWriter readerWriter;
+    int rows = 1000;
```

Comment:
I intentionally wanted to check without chunking in this method, but I added a random chunk size in the next test.